image-spec.csv

```diff
@@ -1,7 +1,7 @@
 context_dir,image_tag,from_tag
 ./base/ubi/9.5-minimal,base-ubi-9.5-minimal
-./golang/1.23,golang-1.23,base-ubi-9.5-minimal
```

Comment:
out of scope, but should we document the steps required to verify a base image is no longer used? I always struggle to be reasonable sure we can drop it